source/components/send-email.ts

```diff
@@ -13,7 +13,7 @@ type Args = {
 export function sendEmail(args: Args): void {
 	const {to = [], cc = [], bcc = [], subject = '', body = ''} = args
 	try {
-		email(to, cc, bcc, subject, body)
+		openUrl(`mailto:${to}?cc=${cc}&bcc=${bcc}&subject=${subject}&body=${body}`)
```

Comment:
We need to make sure that we urlencode these segments

Reply:
Yeah, I agree. I wrote up in the issue that encoding was needed and walked it back incorrectly. I'll try to not muddy the changes too much. Here's at least [how the dependency performed this](https://github.com/anarchicknight/react-native-communications/blob/05bfcdcc123d1e2380156832317a917766699c5c/AKCommunications.js).

Reply:
I think we can use whatever we use to generate query strings elsewhere, and use it to make this one?

Reply:
`to=${encodeUriComponent(to)}?{makeMeQueryString({cc, from, subject}, {omitEmpty: true})}` or something like that